native/cocos/bindings/auto/jsb_scene_auto.cpp

```diff
@@ -20323,6 +20327,42 @@ static bool js_cc_scene_ICameraInfo_trackingType_get(se::State& s)
 }
 SE_BIND_PROP_GET(js_cc_scene_ICameraInfo_trackingType_get) 
 
+static bool js_cc_scene_ICameraInfo_usage_set(se::State& s)
```

Comment:
Don't have to commit binding codes.

Reply:
Sometimes it is hard to separate binding codes, because we have to compile native code on our machines and develop features. So binding codes will usually be committed together with the changes, to make sure the program compiles. There will be several commits before turning into a pr, and at that time, it is hard to revert the binding codes.
After switching to swig, the generated code is stable and will not conflict with auto-generated codes.